isis/src/kaguya/apps/kaguyasp2isis/kaguyasp2isis.xml

```diff
@@ -3,10 +3,13 @@
 <application name="kaguyasp2isis" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://isis.astrogeology.usgs.gov/Schemas/Application/application.xsd">
 
   <brief>
-    Imports a Kaguya SP (Spectral Profiler) file
+    Imports a Kaguya SP (Spectral Profiler) file (deprecated)
```

Comment:
We should list when it will be removed too.

Reply:
@jessemapel  No. Is there an appropriate way to announce or process to go through to indicate that something is deprecated? This is a bit of an odd situation in that the name of the program was changed years ago but the old version was not deleted, so it hasn't actually been supported/updated in some time.

Reply:
Pretty sure it would require a RFC? Do we even have a plan for when/how this should be deprecated?